package-lock.json

```diff
@@ -1474,9 +1474,7 @@
       "version": "3.1.2"
     },
     "lodash": {
-      "version": "4.17.21",
-      "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.21.tgz",
-      "integrity": "sha512-v2kDEe57lecTulaDIuNTPy3Ry4gLGJ6Z1O3vE1krgXZNrsQ+LFTGHVxVjcXPs17LhbZVGedAJv8XZ1tvj5FvSg=="
+      "version": "4.17.20"
```

Comment:
was this downgraded for a reason ?

Reply:
package-lock.json was automatically updated upon running npm install I believe this has to do with the node_modules/lodash version being listed as 4.17.20